django/core/checks/security/base.py

```diff
@@ -182,3 +187,9 @@ def check_xframe_deny(app_configs, **kwargs):
         settings.X_FRAME_OPTIONS == 'DENY'
     )
     return [] if passed_check else [W019]
+
+
+@register(Tags.security, deploy=True)
+def check_allowed_hosts(app_configs, **kwargs):
+    passed_check = not settings.ALLOWED_HOSTS
```

Comment:
@akulakov is right - his comment has been misunderstood as a comment on coding style, when he is actually pointing out that the sense of the test is reversed from what it should be.